content/cloud/linking-applications-to-resources-using-labels.md

```diff
@@ -0,0 +1,143 @@
+
+```bash
+Which database would you like to link to todo-app using the label "data":
+> inspiring-otter
+```
+
+> Note: The database names are arbitrary and created on our behalf. In this case `inspiring-otter` is our Cloud database. We are linking the `inspiring-otter` database to our `todo-app` using the label "data".
+
+From this point, we see that our app is deploying and once finished we receive our deployed application's URL endpoint:
+
+```bash
+Waiting for application to become ready...... ready
+Available Routes:
+  todo-app: https://todo-app-ylnmlqmr.fermyon.app (wildcard)
+```
+
+Let's kick things up a notch and deploy a second Spin application (`todo-app-v2`) that will also use the same Cloud database:
+
+```toml
+// --snip--
+name = "todo-app-v2"
+[component]
+id = "todo"
+sqlite_databases = ["default"]
+// --snip--
+```
+
+When deploying the second Spin application (`todo-app-v2`) we are again asked if we would like to use an existing database and link our application (in this case `todo-app-v2`) to it:
+
+```bash
+spin deploy       
+Uploading todo-app-v2 version 0.1.0-r6e577b18 to Fermyon Cloud...
+Deploying...
+App "todo-app-v2" accesses a database labeled "default"
+Would you like to link an existing database or create a new database?:
+> Use an existing database and link app to it
+  Create a new database and link the app to it
+```
+
+We again use the up/down arrow keys to make our selection and hit `return` to continue:
+
+```bash
+Which database would you like to link to todo-app-mkii using the label "default":
+> inspiring-otter
+```
+
+In the example here we have chosen to link `todo-app-v2` to our pre-existing `inspiring-otter` Cloud database using via the "default" label, that we added to our `todo-app-v2` application manifest (`spin.toml`) file.
+
+Now we have two seperate Spin applications (`todo-app` and `todo-app-v2`) using their own labels to link to the same database. 
+
+Links and labels are application scoped. Whenever a specific Spin application (say `todo-app`) needs to access the database resource (`inspiring-otter`), it will do so via the label "data".
+
+The Spin application does not need to know the true name of the Cloud resource (`inspiring-otter` database) that it is interacting with. 
+
+A label is understood by all the components within a single application, but the labels applied do not hold any significance to other Spin applications. 
+
+> At this time, only NoOps SQL DB supports labels. However, in the future other Fermyon Cloud resources such as Key Value Store will support links and labels as well. 
+
+## Benefits of Links and Labels
+
+Benefits of the Fermyon Clould links and labels include:
+
+* **Easy Sharing**: Share your Fermyon Cloud resource across applications effortlessly.
+* **Resource Creation Control**: You decide when to create and delete Fermyon Cloud resources.
+* **Flexible Lifecycle Management**: Manage Fermyon Cloud resources independently of your Spin application.
```

Comment:
I struggle with this because:

* I can do that with connection strings.
* It doesn't feel very NoOps if I have to manage it.

(The first is what makes this bullet point problematic; the second is grouchy pedantry.)

Reply:
Perhaps this?

```suggestion
* **Decoupled Resource Lifecycle**: Allow Fermyon Cloud resources to have lifecycles independent of your Spin application's lifecycle.
```